### ISSUE TITLE
Is the plog2.0 ready?

### ISSUE THREAD

**Issue by Anubisseu** [iensen/plog2.0#1]
I am a student from Southeast University in Nanjing, China. I want to do some experiments on p-log. 
Is the plog2.0 ready for usage?

**Comment by iensen** [comment]
Hi, Anubisseu.

I am excited to see you are following my repository!:) 
Unfortunately, it's not ready yet. I am working on it...

You may try this one: http://www.depts.ttu.edu/cs/research/krlab/plog.php
We are aware of some errors in it though...

This version, once it's finished is intended to be much more user-friendly with better error handling, and hopefully more efficient for a large class of programs:)

Thanks.

**Comment by Anubisseu** [comment]
Thanks for your reply, I have tried the P-log 1.0, but it seems dose not support cr-rule. What a pity! :sob:

**Comment by iensen** [comment]
You're right, unfortunately the first version does not support cr-rules...It also has poor type and error checking, which makes it difficult to use.

I think all  Plog algorithms I am aware of can be easily modified to support cr-rules, so we will have them sooner or later!

I hope to get something working by the end of the year...

**Comment by Anubisseu** [comment]
Anyway, thanks for your reply angin!

**Comment by domoritz** [comment]
I'm going to subscribe to this issue as well. Looking forward to a working version.
